src/public/css/partials/footer.css

```
@@ -0,0 +1,111 @@
+footer {
+    background-color: #141718;
+    color: #fff;
+    padding: 10px 100px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-size: 15px;
+}
+a{
+    color: #fff;
+    text-decoration: none; 
+}
+.div-footer-1{
+    display: grid;
+    grid-template-columns: 200px 250px 80px 80px 80px 80px;
+    grid-template-rows: 150px;
+    align-items: center;
+
+}
+.div-footer-2{
+    display: grid;
+    grid-template-columns: 270px 400px 70px;
+    grid-template-rows: 100px;
+    align-items: center;
+    border-top:1px solid #444;
```

Esto va en una variable.
```suggestion
    border-top:1px solid var(--asdfasdf);
```